appdatabase/migrations/sql/0017_bookmarks.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS bookmarks (
+  url VARCHAR NOT NULL,
+  name BIGINT NOT NULL,
```

Comment:
That's interesting, it clearly works as it's tested, is it intentional to have name as BIGINT and not a VARCHAR?

Reply:
Oops, not at all. Copypaste error. I'll fix